pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -317,7 +317,8 @@ public List<MercuryOrderDto> getByOrderId(String orderId) {
                                 .withStatusDate(rs.getLong(DBConstants.MERCURY_STATUS_DATE))
                                 .withOrderStatus(rs.getString(DBConstants.MERCURY_ORDER_STATUS))
                                 .withDdpParticipantId(rs.getString(DBConstants.DDP_PARTICIPANT_ID))
-                                .withTissueId(rs.getInt(DBConstants.TISSUE_ID))
+                                .withTissueId(rs.getObject(DBConstants.TISSUE_ID) != null
```

Comment:
Here's where sql null turns into 0, leading to downstream query failures when looking for a nonexistent tissue with id 0.